concepticondata/conceptlists/Zalizniak-2020-2591.tsv

```diff
@@ -0,0 +1,2592 @@
+Zalizniak-2020-2591-97	97	slow (adj.)	SLOW	701	97:→:2 97:—:2 97:—:2 97:—:2 97:—:1 97:→:1 97:—:1 97:—:1 97:—:1 97:—:1	shift1822 shift3055 shift3056 shift3941 shift2808 shift1901 shift1286 shift2992 shift2031 shift3050	10	14
+Zalizniak-2020-2591-98	98	left	LEFT	244	98:—:6 98:↔:4 98:→:3 98:→:3 98:—:2 98:→:2 98:→:1 98:→:1 98:→:1 98:—:0	shift918 shift905 shift937 shift2866 shift909 shift906 shift3340 shift3428 shift2847 shift924	10	23
+Zalizniak-2020-2591-99	99	forehead	FOREHEAD	123	99:—:4 99:→:3 99:—:2 99:→:2 99:→:2 99:—:1 99:→:1 99:→:1 99:—:1 99:→:1	shift540 shift296 shift2815 shift527 shift2331 shift1778 shift2332 shift528 shift897 shift3571	10	18
+Zalizniak-2020-2591-100	100	firm, solid	SOLID	3003	100:→:10 100:—:6 100:—:5 100:→:5 100:→:3 100:↔:2 100:—:2 100:—:2 100:—:1 100:→:1	shift2480 shift717 shift718 shift1222 shift1359 shift2497 shift719 shift1933 shift672 shift720	10	37
+Zalizniak-2020-2591-101	101	fire	FIRE	221	101:—:46 101:→:13 101:→:12 101:↔:7 101:→:4 101:—:3 101:→:2 101:→:2 101:→:1 101:→:1	shift3852 shift2366 shift3859 shift3082 shift3266 shift4093 shift3728 shift151 shift3440 shift3108	10	91
+Zalizniak-2020-2591-102	102	clever	CLEVER	1310	102:—:4 102:→:3 102:—:2 102:—:2 102:—:1 102:—:1 102:→:1 102:—:1 102:—:1 102:→:1	shift1245 shift3458 shift135 shift2549 shift101 shift2809 shift2573 shift2258 shift2780 shift3467	10	17
+Zalizniak-2020-2591-103	103	clean (adj.)	CLEAN	704	103:→:5 103:—:4 103:→:4 103:—:3 103:→:2 103:→:2 103:→:2 103:—:1 103:—:1 103:—:1	shift213 shift768 shift1922 shift3934 shift2898 shift2311 shift2268 shift2958 shift2904 shift4123	10	25
+Zalizniak-2020-2591-104	104	beautiful	BEAUTIFUL	1427	104:→:15 104:↔:4 104:—:2 104:—:2 104:—:1 104:—:1 104:—:1 104:—:1 104:—:1 104:—:1	shift392 shift413 shift1998 shift3822 shift1558 shift375 shift3696 shift4002 shift4267 shift3783	10	29
+Zalizniak-2020-2591-105	105	to smear, anoint	SMEAR	364	105:→:16 105:→:5 105:→:3 105:→:2 105:→:2 105:→:1 105:→:1 105:→:1 105:→:1	shift1107 shift1105 shift2884 shift2890 shift903 shift1106 shift1412 shift2365 shift2986	9	32
+Zalizniak-2020-2591-106	106	to sit	SIT	1416	106:→:10 106:→:5 106:→:2 106:→:2 106:—:1 106:—:1 106:→:1 106:→:1 106:—:1	shift756 shift2472 shift3627 shift1620 shift3213 shift3626 shift4070 shift229 shift3746	9	24
+Zalizniak-2020-2591-107	107	to search, to look for	LOOK FOR	1468	107:→:21 107:→:19 107:→:15 107:↔:8 107:—:3 107:—:1 107:—:1 107:—:1 107:—:1	shift99 shift9 shift2190 shift750 shift658 shift4215 shift3615 shift4301 shift1454	9	70
+Zalizniak-2020-2591-108	108	to run	RUN	1519	108:↔:12 108:↔:5 108:→:3 108:→:2 108:→:1 108:→:1 108:→:1 108:→:1 108:→:1	shift1632 shift685 shift1306 shift1704 shift3399 shift2924 shift1232 shift3229 shift2925	9	27
+Zalizniak-2020-2591-109	109	to reach	ARRIVE	1387	109:→:9 109:→:6 109:→:5 109:→:4 109:→:3 109:→:2 109:→:2 109:—:1 109:→:1	shift1066 shift1764 shift25 shift1766 shift1765 shift1767 shift1621 shift2566 shift1622	9	33
```

Comment:
This is a tricky one. There are two glosses _to reach_ in the database and it seems that this one has the taxon Motion (http://datsemshift.ru/meanings/t). But I'll unmap it because the shifts do not match the "arrive" meaning.